packages/batch/tests/unit/processPartialResponse.test.ts

```diff
@@ -36,7 +36,7 @@ describe('Function: processPartialResponse()', () => {
     context,
   };
 
-  const handlerWithSqsEvent = async (
+  const handlerWithSqsEvent = (
```

Comment:
While we aren't actually using `await` in this function, it is still useful information to know that this function returns a promise so can we add a return value to the type signature here?

Reply:
Fair, added the `Promise<PartialItemFailureResponse>` return type